indy_common/constants.py

```diff
@@ -160,6 +171,10 @@
 GET_REVOC_REG_DELTA = IndyTransactions.GET_REVOC_REG_DELTA.value
 CHANGE_KEY = IndyTransactions.CHANGE_KEY.value
 
+# Rich Schema
+SET_CONTEXT = IndyTransactions.SET_CONTEXT.value
```

Comment:
Out general pattern for write requests (transaction) is a Noun: NYM, SCHEMA, CLAIM_DEF. etc.
So, I think we may rename `SET_CONTEXT` to `CONTEXT` for consistency.
Also I think that `CONTEXT` may sound too generic. Maybe we can call it `SCHEMA_CONTEXT`?

Reply:
We can't call it `SCHEMA_CONTEXT` because context objects may be used by more than just schemas.
As far as changing it from `SET_CONTEXT` to simply `CONTEXT` I believe @ken-ebert had good reason for `SET_CONTEXT` and perhaps could add some comments here.